cfme/networks/views.py

```diff
@@ -442,7 +443,11 @@ class NetworkRouterAddView(BaseLoggedInPage):
     cloud_tenant = Select(name='cloud_tenant_id')
     add = Button('Add')
 
-    is_displayed = displayed_not_implemented
+    @property
+    def is_displayed(self):
+        return (super(BaseLoggedInPage, self).is_displayed and
```

Comment:
Please use `self.logged_in_as_current_user` directly, since that's the only real check that we want from `BaseLoggedInPage` anyway.